test/integration/main_test.go

```diff
@@ -217,6 +242,19 @@ contacts:
   email:
   - group@example.com
 `, tc.resourceName, traceID.String()))
+			} else {
```

Comment:
This if else is a little bit ugly. Usually you need to do refactor to avoid it.

Reply:
This is removed since now we add subscope to existing test cases.

Reply:
You don't want to add subscope for all test cases including failure cases, right?